package.json

```diff
@@ -23,6 +24,7 @@
     "react-dom": "17.0.2",
     "react-katex": "^2.0.2",
     "react-markdown": "^8.0.1",
+    "react-tweet-embed": "^2.0.0",
```

Comment:
I don't see this being used but perhaps you are planning in using this in the coming features. Just wanted to point it out.